medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,17 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_primary_id = fields.Many2one(
```

Comment:
Yes it is just to add the parent_id into medical_center_ids.

Inverse is not needed if it's stored.